FoodPin/FoodPin/Model/Restaurant.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Restaurant.swift
+//  FoodPin
+//
+//  Created by Andreea Sburlea on 4/19/22.
+//
+
+struct Restaurant: Hashable {
+    var name: String = ""
```

Comment:
These properties should be private and have public getters. You can talk to Vivi about how she did it